test/common/protobuf/descriptor_test.cc

```diff
@@ -19,10 +19,6 @@ namespace Envoy {
 // services as well as types that are referenced in Any messages. IF THIS TEST BREAKS YOU
 // HAVE DONE SOMETHING BAD. Consult with the larger dev team on how to handle.
 TEST(ProtoDescriptorTest, BackCompat) {
-  // Hack to force linking of the service: https://github.com/google/protobuf/issues/4221
```

Comment:
I'm confused. Without linking to grpc_mux stuff, how does this work now?

Reply:
You are right, I need it in two places :(